.tito/packages/mock-core-configs

```diff
@@ -1 +1 @@
-34.6-1 mock-core-configs/
+35-1 mock-core-configs/
```

Comment:
This should not be part of this change. This is done automatically when you do `tito tag` which is part of release process.